### PR TITLE
Blocitoff Sign in and Sign out

### PR DESCRIPTION
**Pull request by ralexr22**
I wanted to dress it up a little, didn't get much further than changing the header.  The "getting started" with devise had me this far, so I didn't have to change much of the code today.

### CODE REVIEW

**Comment by ralexr22** [comment]
I'm so used to `push` and being done, that I didn't realize my terminal wanted me to type in my password.  So let me know if this pull request is incomplete.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -3,14 +3,26 @@
 <head>
   <title>Blocitoff</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>`
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
-<body>
+<body> body { padding-top:90px; }
```
Comment (alexstophel) [inline app/views/layouts/application.html.erb:10]: The `body { padding-top:90px; }` should be in a stylesheet.

**Comment by alexstophel** [comment]
👍 Other than my one comment,

**Comment by ralexr22** [comment]
Ha, I don't even remember why I left that in there, but by the time I got to the next checkpoint, I had removed it.

**Comment by ralexr22** [comment]
Do I delete the branch now that it's merged, or is github just letting me know I can if I want to.  What's standard practice?

**Comment by alexstophel** [comment]
Yep! Go ahead and delete it.

**Pull request merged**